hub/saml-engine/src/integration-test/java/uk/gov/ida/integrationtest/hub/samlengine/apprule/support/SamlEngineAppRule.java

```diff
@@ -59,7 +59,7 @@
 import static uk.gov.ida.saml.metadata.ResourceEncoder.entityIdAsResource;
 
 public class SamlEngineAppRule extends DropwizardAppRule<SamlEngineConfiguration> {
-    private static final String VERIFY_METADATA_PATH = "/uk/gov/ida/saml/metadata/federation";
+    public static final String VERIFY_METADATA_PATH = "/uk/gov/ida/saml/metadata/federation";
```

Comment:
This isn't a great way of doing it, but its consistent so lets keep it for now. It would probably be worth moving the config out to one place, rather than it living at the top of a class that delivers functionality.